cloudflare/resource_cloudflare_ip_list.go

```diff
@@ -0,0 +1,170 @@
+package cloudflare
+
+import (
+	"context"
+	"fmt"
+	"github.com/cloudflare/cloudflare-go"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+	"github.com/pkg/errors"
+	"regexp"
+)
+
+func resourceCloudflareIPList() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceCloudflareIPListCreate,
+		Read:   resourceCloudflareIPListRead,
+		Update: resourceCloudflareIPListUpdate,
+		Delete: resourceCloudflareIPListDelete,
+		Importer: &schema.ResourceImporter{
+			State: resourceCloudflareIPListImport,
+		},
+
+		Schema: map[string]*schema.Schema{
```

Comment:
Do we need `account_id` in here somewhere to tell the provider which accont to operate one? Or do you have a different setup in mind?

Reply:
I'm not having a strong opinion on this. I would have reused the account_id from the provider block here since it is configuring the API client already. On the other hand I can see account_id being configured on the resource-level for some Access related things. (`cloudflare_access_service_tokens` for example). What do you think?

Reply:
Just read your last comment, Multi-Account usage makes totally sense. I've added this to the resource.